.dockerignore

```diff
@@ -15,8 +15,8 @@
 **/bin
 **/charts
 **/docker-compose*
+**/compose*
 **/Dockerfile*
-**/node_modules
```

Comment:
This could make docker context quite bigger, but ignoring node_modules will trigger installation every time we start the container.

Reply:
Fixed in #6